utils/BaseStore.js

```diff
@@ -1 +1,4 @@
-module.exports = require('dispatchr/utils/BaseStore');
+// @TODO deprecate
+//console.warn("require('fluxible/utils/createStore') is deprecated. Please use " +
```

Comment:
Oh I see. If we're going to do it, warnings won't hurt. And the sooner users see it, the better, right?